ui/css/treeherder.css

```diff
@@ -213,8 +213,13 @@ th-watched-repo {
 
 #platform-job-text-search-field {
     height:28px;
-    width: inherit;
+    width: 150px;
     display: inherit;
+    transition: width .2s;
+}
+
+#platform-job-text-search-field:focus,#platform-job-text-search-field:valid {
```

Comment:
Maybe we might want to wrap this line on the first comma, it seems to be the convention in treeherder.css.
```
#platform-job-text-search-field:focus,
#platform-job-text-search-field:valid {
```